crates/holochain_sqlite/src/db/access.rs

```diff
@@ -453,7 +453,7 @@ impl<Kind: DbKindT + Send + Sync + 'static> DbWrite<Kind> {
             kind,
             PoolConfig::default(),
             Some(|sql| {
-                println!("SQL: {}", sql);
+                tracing::debug!("SQL: {}", sql);
```

Comment:
Instead of always outputting debug info to stdout, use tracing which can be enabled.

Reply:
Dumping all sql statements run is literally tracing.

```suggestion
                tracing::trace!("SQL: {}", sql);
```